scripts/mattermost-mysql-5.0.sql

```diff
@@ -369,7 +369,7 @@ CREATE TABLE `IncomingWebhooks` (
   `DisplayName` varchar(64) DEFAULT NULL,
   `Description` varchar(128) DEFAULT NULL,
   `Username` varchar(255) DEFAULT NULL,
-  `IconURL` varchar(255) DEFAULT NULL,
+  `IconURL` text,
```

Comment:
Changing this file is probably incorrect: it represents the schema of the v5.0 database at the time /that/ code existed, and it's used to verify that the migration /from/ this schema to the current schema is correct. See `scripts/mysql-migration-test.sh` (and same for Postgres).

Reply:
Oh I see. Reverting...